### PR TITLE
[EHI - 50] Function to find all elements greater than all elements to its right

### PR DESCRIPTION
**Pull request by RijviZ**

### CODE REVIEW

File: src/main/data_structure/stack.py [hunk 1]
```diff
@@ -39,3 +44,22 @@ def size(self):
     # to see elements of stack
     def display(self):
         return self.stack
+
+    # Function to find all elements which are greater than all elements present to its right
```
Comment (jonyroy) [inline src/main/data_structure/stack.py]: implement this with List instead of Stack

**Review by jonyroy — CHANGES_REQUESTED** [review]

File: src/main/data_structure/list_exercise.py [hunk 2]
```diff
@@ -112,7 +111,8 @@ def binary_search_target_index(list_of_number, value, first_index):
             # sorted list .[0,1,2,2,2,3,4,5] our target value = 2 and we
             # looking for the first indexed 2 or last indexed 2 in the list
             if first_index:
-                end = mid - 1 
+                end = mid - 1
+
```
Comment (imShakil) [inline src/main/data_structure/list_exercise.py:133]: we can remove this space

File: src/main/data_structure/list_exercise.py [hunk 3]
```diff
@@ -138,26 +135,28 @@ def binary_search_count(list_of_number, value):
 def binary_search_circular_sorted(list_of_numbers, value):
     start = 0
     end = len(list_of_numbers) - 1
-    
+
```
Comment (imShakil) [inline src/main/data_structure/list_exercise.py:156]: we can remove this space

File: src/main/data_structure/list_exercise.py [hunk 4]
```diff
@@ -138,26 +135,28 @@ def binary_search_count(list_of_number, value):
...
     while start <= end:
-        mid = start + (end-start) // 2 
-        
+        mid = start + (end - start) // 2
+
         if list_of_numbers[mid] == value:
             return 'Yes'
-        
+
+
```
Comment (imShakil) [inline src/main/data_structure/list_exercise.py:164]: We can remove this space

File: src/main/data_structure/list_exercise.py (hunk 2 above)
Comment (RijviZ) [reply src/main/data_structure/list_exercise.py:133]: I did not solve this.
This is not mine

**Review by jonyroy — APPROVED** [review]

**Pull request merged**